.changeset/three-falcons-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@clerk/express": patch
```

Comment:
I think minor is more appropriate
```suggestion
"@clerk/express": minor
```

Reply:
Updated to `major` since Express v5 tagged as `latest` will come soon anyway and that we can start deprecating `@clerk/clerk-node-sdk` too. What y'all think?

cc @nikosdouvlis